pages/search.js

```diff
@@ -143,15 +143,15 @@ class SearchPage extends React.Component {
 
   render() {
     const { data, term = '', intl } = this.props;
-    const { error, loading, search } = data || {};
+    const { error, loading, accounts } = data || {};
 
     if (error) {
       return <ErrorPage data={this.props.data} />;
     }
 
     const filters = ['ALL', 'COLLECTIVE', 'EVENT', 'ORGANIZATION', 'HOST'];
-    const { collectives, limit = 20, offset, total = 0 } = search || {};
-    const showCollectives = term.trim() !== '' && !!collectives;
+    const { nodes: accountNodes, limit = 20, offset, totalCount = 0 } = accounts || {};
```

Comment:
accountNodes doesn't looks good. you can just use `accounts.nodes` when you need it.